src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -408,6 +420,10 @@
                     window.clearInterval(this.pollingId);
                 }
             },
+        updateDatasetVersion(releaseId, enableImport){
```

Comment:
For `selectDataset` to be able deduce for itself, it would need some knowledge of whether `useRelease` or `useData` is selected in the UI. If it only received a `null` or `undefined` `datasetVersionId` from `selectRelease`, would it be able to distinguish on its own whether that was because no release had been selected or because `useData` was selected and the releases tree select was disabled?

Reply:
I see now, sorry. So `valid` or something similar might be better than `enableImport`. But maybe the child component should instead emit two events: `valid` events (to indicate wther the form is valid), and `releaseId` values (only emitted when the form is valid). It doesn't feel quite right that this component is overwriting its `newDatasetVersionId` version even when the child form isn't in a valid state.